integration-test/common/src/test/java/com/tencent/rss/test/ShuffleServerWithMemLocalHdfsTest.java

```diff
@@ -39,10 +39,7 @@
 import com.tencent.rss.storage.handler.impl.MemoryQuorumClientReadHandler;
 import com.tencent.rss.storage.util.StorageType;
 
-import org.junit.After;
-import org.junit.Before;
-import org.junit.BeforeClass;
-import org.junit.Test;
+import org.junit.*;
```

Comment:
don't use * in import, it should be included in rule check

Reply:
Fixed